### DISCUSSION TITLE
EDGETX BUG, the mixed empty channel cannot be deleted

### DISCUSSION THREAD

**Discussion by wsnfly** [ExpressLRS/ExpressLRS#1373] [Q&A] [answered]
I wanted to test some functions of the system logic switch today, and set the test output on channels 31 and 32, but when I wanted to delete them, I found that they could not be deleted or replaced. Developers please take a look.

https://www.youtube.com/watch?v=bUFw3TmUSF0

**Comment by AlessandroAU** [comment, marked as answer]
You should post this on the EdgeTX github, not here.
